test/e2e/config/vsphere.yaml

```diff
@@ -26,6 +26,15 @@ providers:
   - name: cluster-api
     type: CoreProvider
     versions:
+      - name: "v1.9.99"
```

Comment:
Wondering if we have to make changes to our clusterctl upgrade tests (happy to pair on it later to figure it out)

Reply:
adjusted, it already is then always updating to v1.9.99 :-) so only adjusted the names of the tests to be correct

Reply:
Sounds good, will take another look (likely Monday though)